docs/fundamentals/typography.md

```diff
@@ -30,48 +30,69 @@ Headings, subheadings, and body text each serve distinct functions and should be
 
 ## Design ---
 
-Element defines 9 different type styles. It follows the
-HTML semantic, while omitting the `H4`, `H5` and `H6` heading to reduce visual
-complexity of titles within an application.
-
-### Type scale
-
-Line heights are computed using `font-size + 1/8rem` (used in buttons,
-navigation, tables…).
-
-Type             | Size (rem) | Line Height (rem) | Family       | Style
------------------|------------|-------------------|--------------|------
-Data Callout     | 3          | 3.125             | Siemens Sans | Roman
-H1 Heading       | 1.85       | 2                 | Siemens Sans | Roman
-H1 Heading Black | 1.85       | 2                 | Siemens Sans | **Black**
-H2 Subheading    | 1.25       | 1.5               | Siemens Sans | Roman
-H3 Subheading    | 1.125      | 1.25              | Siemens Sans | **Bold**
-Title 1          | 1          | 1.125             | Siemens Sans | **Bold**
-Body 1           | 1          | 1.125             | Siemens Sans | Roman
-Title 2          | 0.875      | 1                 | Siemens Sans | **Bold**
-Body 2           | 0.875      | 1                 | Siemens Sans | Roman
-Caption          | 0.75       | 1                 | Siemens Sans | Roman
-Display 1        | 3.5        | 4.5               | Siemens Sans | Roman
-Display 2        | 3          | 4                 | Siemens Sans | Roman
-Display 3        | 2.5        | 3.25              | Siemens Sans | **Black**
-Display 4        | 2.5        | 3.25              | Siemens Sans | Roman
-
-All font sizes are defined in **rem** (the root em size) based on the root
-element size. The Element Design System defines **1 rem as 16 px** while
-also applying this fixed conversion ratio within the implementation.
-
-### Applying the type scale
-
-- Choose **Heading 1** or **Heading 2** as the main title based on spacing
-  requirements. If using Heading 1, follow it with Heading 2 for major
-  section titles.
-- **H3 subheading** serves as a secondary heading.
-- **Title 1** and **Title 2** are used for cards or less important section titles.
-- Use body styles within components and text blocks, with **Body 2** as the default.
-- Use **Caption** for small, supplementary text providing additional context.
-- Use **Display** styles in widgets, key visual elements, hero sections or any area where more prominence is required.
-
-![Typography styles](images/typography-styles.png)
+Element’s typography is divided into body, headings and display styles.
+
+### Body
+
+Body text is used for the main content, often following a heading or as part of UI components.
+
+- Use `body` as the default style for most components and text blocks.
+- Use `body-lg` for larger text and can be used for more prominent text.
```

Comment:
That sentence here sounds weird since a mic of a description and a command. WDYT about:

```suggestion
- Use `body-lg` for larger and more prominent text.
```